cmd/evm/internal/t8ntool/flags.go

```diff
@@ -30,17 +30,17 @@ var (
 		Name:  "trace",
 		Usage: "Output full trace logs to files <txhash>.jsonl",
 	}
-	TraceDisableMemoryFlag = cli.BoolFlag{
-		Name:  "trace.nomemory",
-		Usage: "Disable full memory dump in traces",
+	TraceEnableMemoryFlag = cli.BoolFlag{
```

Comment:
You can preserve this flag `TraceDisableMemoryFlag` but change the type as `cli.TBoolFlag`. So that the default value is True. I think we should keep the compatibility if possible.

Reply:
I agree with this for all flags.

 (but please keep the existing change to `LogConfig`)